basic_pipeline/1_Introduction.md

```diff
@@ -0,0 +1,55 @@
+# Introduction
```

Comment:
I think it is worth mentioning, that since membrane is written in Elixir some knowledge of it is required - we could link to Elixir's guide page for reference.

Reply:
done